packages/relay/src/Environment.js

```diff
@@ -1,19 +1,7 @@
 // @flow
 
-import { createEnvironment } from '@mrtnzlml/relay';
-import { inMemoryFetcher } from '@kiwicom/margarita-graphql';
-
-const createInMemoryFetcher = () => {
-  return function(request, variables, uploadables) {
-    if (uploadables) {
-      throw new Error(
-        'Uploadables are not supported in this in-memory Relay environment.',
-      );
-    }
-    return inMemoryFetcher(request.text, variables);
-  };
-};
+import { createEnvironment, createNetworkFetcher } from '@mrtnzlml/relay';
 
 export default createEnvironment({
-  fetcherFn: createInMemoryFetcher(),
+  fetcherFn: createNetworkFetcher('http://127.0.0.1:4000'),
```

Comment:
Maybe this should point to an ENV variable instead? If we were to deploy this to a server, it would not work